src/worker_loader.js

```diff
@@ -15,21 +15,20 @@
 
 'use strict';
 
-if (typeof PDFJSDev === 'undefined' || !PDFJSDev.test('PRODUCTION')) {
```

Comment:
It's needed only for requirejs (and probably for SystemJS as well) -- our tests were failing without it. Since we are not using src/worker_loader.js  in the production anyway, I decided to remove the pre-processor from here.